### DISCUSSION TITLE
<title> Lost access to Github Copilot

### DISCUSSION THREAD

**Discussion by tiany7** [community/community#43642] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi! I am a student of Columbia Unviersity. I signed up for the student package on October and gained access to the Github Copilot. 
However, I lost access to the github copilot today, while I am still having a pro package. It seems that the only way to enable it is to purchase it and start billing. Could you help me to figure this out?

**Comment by otkrickey** [comment]
I have created a discussion for today's case. Please share your situation. #43634

**Comment by otkrickey** [comment]
Solved: #43634
